### PR TITLE
ROADMAP: Name change

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses ROADMAP

Updates the name of the package to pysatMissions.  Need to change this now before we have too many users.

## Type of change

- Breaking change (fix or feature that would cause existing functionality
      to not work as expected)

# How Has This Been Tested?

On Travis CI

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: pysatMissions/tests/test_methods_aacgmv2.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 import pysat
-import pysatMissionPlanning.methods.aacgmv2 as methaacgm
+import pysatMissions.methods.aacgmv2 as methaacgm
```
Comment (aburrell) [inline pysatMissions/tests/test_methods_aacgmv2.py]: or just `aacgmv2`

File: pysatMissions/tests/test_methods_apexpy.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 import pysat
-import pysatMissionPlanning.methods.apexpy as methapex
+import pysatMissions.methods.apexpy as methapex
```
Comment (aburrell) [inline pysatMissions/tests/test_methods_apexpy.py]: The best high brings you up to the apex.

File: pysatMissions/tests/test_methods_pyglow.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 import pysat
-from pysatMissionPlanning.methods import pyglow as methglow
+from pysatMissions.methods import pyglow as methglow
```
Comment (aburrell) [inline pysatMissions/tests/test_methods_pyglow.py]: `mm_glow`

File: pysatMissions/tests/test_methods_spacecraft.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 import pysat
-from pysatMissionPlanning.methods import spacecraft as methsc
+from pysatMissions.methods import spacecraft as methsc
```
Comment (aburrell) [inline pysatMissions/tests/test_methods_spacecraft.py]: `mm_sc`

File: README.md [hunk 5]
```diff
@@ -68,7 +68,7 @@ The methods that run empirical models can also be exported to any pysat instrume
 
 ```
 import pysat
-import pysatMissionPlanning.methods.pyglow as methglow
+import pysatMissions.methods.pyglow as methglow
```
[the remark this answers is not in the document]
Comment (jklenzing) [reply README.md]: fixed

File: pysatMissions/instruments/pysat_ephem.py [hunk 6]
```diff
@@ -39,10 +39,10 @@ def init(self):
...
 
     """
-    import pysatMissionPlanning.methods.aacgmv2 as methaacgm
-    import pysatMissionPlanning.methods.apexpy as methapex
-    import pysatMissionPlanning.methods.pyglow as methglow
-    import pysatMissionPlanning.methods.spacecraft as methsc
+    import pysatMissions.methods.aacgmv2 as methaacgm
+    import pysatMissions.methods.apexpy as methapex
+    import pysatMissions.methods.pyglow as methglow
+    import pysatMissions.methods.spacecraft as methsc
```
[the remark this answers is not in the document]
Comment (jklenzing) [reply pysatMissions/instruments/pysat_ephem.py]: fixed

**Comment by jklenzing** [comment]
All the methods should be imported as mm_* now, with the exception of mcore.

**Pull request merged**